src/main/resources/component-kernel.conf

```diff
@@ -319,10 +319,19 @@ docker {
 
     # Can be used to supply a custom host IP. This might be useful
     # when the environment is composed within a container (e.g. on a build system).
-    #
+    # It will supply the docker-compose environment
+    # with the value as 'DOCKER_HOST' and use the IP for internal addressing.
+    # 'DOCKER_HOST' will be prepended with TCP protocol and appended with #docker.hostPort.
+    # E.g.: DOCKER_HOST = "tcp://<IP>:2375"
+    # If left empty, docker-compose will default to local docker daemon via socket
+    # (like unix:///var/run/docker.sock Note: that is no valid configuration as IP is missing).
     hostIp = ""
     hostIp = ${?SIRIUS_DOCKER_HOST}
 
+    # Used in combination with #docker.hostIp to determine the 'DOCKER_HOST' TCP address.
+    hostPort = "2375"
+    hostPort = ${?SIRIUS_DOCKER_HOST}
```

Comment:
is this the right key? it's the same as for hostIp

Reply:
Probably not

Reply:
thx